java/arcs/core/storage/StorageProxy.kt

```diff
@@ -353,6 +370,18 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T>(
         )
     }
 
+    private fun sendMessageToStore(
+        message: ProxyMessage<Data, Op, T>,
+        deferred: CompletableDeferred<Boolean> = CompletableDeferred()
+    ) {
+        val queueNum = outgoingMessagesInFlight.getAndIncrement()
+        log.debug { "Queueing message (pos: $queueNum) for sending to the store: $message" }
+        if (queueNum == 0) {
+            busySendingMessagesChannel.offer(true)
+        }
+        outgoingMessagesChannel.offer(message to deferred)
```

Comment:
Where's the actual sending to store logic now? I don't see store.onProxyMessage being called anymore.

Reply:
Ah I see it now, that was added previously and lives in the init block.

Reply:
Line 89 in the outgoingMessages flow onEach()